services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java

```diff
@@ -826,7 +844,7 @@ private boolean isFlowActive(FlowPairDto<FlowDto, FlowDto> flowPair) {
     }
 
     private void initFlowCache() {
-        PathComputerFlowFetcher flowFetcher = new PathComputerFlowFetcher(pathComputer);
+        BidirectionalFlowFetcher flowFetcher = new BidirectionalFlowFetcher(repositoryFactory.createFlowRepository());
```

Comment:
Same issue with `InvalidArgumentException`.

Reply:
Not an issue. We catch and log IllegalArgumentException.